__tests__/app.test.js

```diff
@@ -30,3 +30,54 @@ describe("1. GET /api/topics", () => {
       });
   });
 });
+
+describe("2. GET /api/articles", () => {
+  test("200: should respond with an array of article objects", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: articles }) => {
+        const articlesArr = articles.articles;
+        expect(articlesArr).toHaveLength(12);
+        articlesArr.forEach((article) => {
+          expect(article).toEqual(
+            expect.objectContaining({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              comment_count: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+  it("200: should return articles sorted by date descending order by default", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(
+          articles.every((article, index) => {
```

Comment:
there is a package could jest-sorted which will help you check the order in which articles come back too. This will allow you to check more than just created at in the future